components/sidebar/index.css

```diff
@@ -26,3 +26,43 @@
 .close:after {
   transform: rotate(-45deg);
 }
+
+ul.sidebar-navigation,
+ul.sidebar-navigation li > a {
```

Comment:
I would prefer use class to style this. It's more convenient. If you then go to `<button>` it would be saver.